### DISCUSSION TITLE
Retrieving bearer token (using vault)

### DISCUSSION THREAD

**Discussion by robert-lor** [prometheus/prometheus#11237] [Q&A]
It is possible in prometheus to retrieve secrets from vault and using them for retrieving bearer token? How? Thank you.

**Comment by roidelapluie** [comment]
https://inuits.eu/blog/prometheus-consul-vault-228/

**Reply by robert-lor** [reply]
Thak you. I already sow this article. A problem is that I have to retrieve token (with secrets from vault) using a specific endpoint - and not from vault.

**Reply by roidelapluie** [reply]
You could use wget and store the token to a file
